docs/templates/article.html

```diff
@@ -66,6 +96,19 @@ <h3 class="mb-1">5. Give it time to do its thing</h3>
 						<p class="mb-3">A hair masque is not the same as shampoo or conditioner – it needs a bit more time to penetrate the strands and repair from the inside out. So, for silky soft results, apply your hair masque to towel dried hair and leave on for a minimum of 5 to 10 minutes before rinsing. Leave it longer if you have time on your hands.</p>
 						<p class="mb-3">Got thirsty hair in dire need of some serious TLC? You can also apply our nourishing fig and coconut masque to dry hair and let it soak in for at least 20 mins. Or for butter-soft results, pop on a shower cap, silk scarf or our super-cute, static-free Microfibre Hair Towel Wrap and sleep in your hair masque overnight!</p>
 					</div>
+					
+					<div class="mb-1">23 April, 2020</div>
+					<ul class="list-inline mb-4 mt-1">
+						<li class="list-inline-item">
+							<a href="#" class="text-decoration-none sni sni__twitter-square text-primary social-icon-primary"></a>
+						</li>
+						<li class="list-inline-item">
+							<a href="#" class="text-decoration-none sni sni__facebook-square text-primary social-icon-primary"></a>
+						</li>
+						<li class="list-inline-item">
+							<a href="#" class="text-decoration-none sni sni__pinterest-square text-primary social-icon-primary"></a>
+						</li>
+					</ul>
 				</div>
 
```

Comment:
please review the integration of the section SIDEBAR as it's done in compounds, we must use the same code adding additional classes for the position. sticky-bottom__container is not used as also .d-lg-flex. Please review all the classes used and remove the class and code not used.

Reply:
I am not understanding this one, also I am not really sure what has been done on the compound?
the sticky-bottom__container is used to make it sticky